Source/WebCore/platform/network/soup/AuthenticationChallengeSoup.cpp

```diff
@@ -90,7 +90,7 @@ AuthenticationChallenge::AuthenticationChallenge(SoupMessage* soupMessage, SoupA
 {
 }
 
-static ProtectionSpace protectionSpaceForClientCertificate(const URL& url)
+static ProtectionSpace AuthenticationChallenge::protectionSpaceForClientCertificate(const URL& url)
```

Comment:
You have to remove the static here, does this build?

Reply:
Yup, I know, wanted to be a bit too quick. It's evening here, so the rest will be for tomorrow.